docs/decex/fraud_proofs.md

```diff
@@ -164,10 +162,10 @@ Upon receiving the proof, the consensus node can rerun the shuffle algorithm to
 
 - `domain_id`: ID of the domain this fraud proof targets.
 - `bad_receipt_hash`: the targeted invalid ER.
-- `mmr_proof`: MMR proof for the parent consensus block of the receipt.
+- `mmr_proof`: MMR proof for the consensus block containing the receipt.
```

Comment:
Actually, it is the MMR proof for the consensus block that derives the ER, because there are some states in this block used during the ER derivation so we need the MMR proof to get the state root of this block and used to verify the storage proof of the these states.